code/controllers/configuration/entries/game_options.dm

```diff
@@ -372,7 +372,7 @@
 	min_val = 0
 
 /datum/config_entry/number/space_budget
-	config_entry_value = 16
+	config_entry_value = 60
```

Comment:
```suggestion
	config_entry_value = 40
```

Maploading is still expensive even if it doesn't OOM us. Tweak the value and I'll merge.